iambic/config/wizard.py

```diff
@@ -997,6 +997,14 @@ def setup_aws_configuration(self):
             self.hub_account_id = None
 
         try:
+            if getattr(self, "boto3_session", None) is None:
```

Comment:
Nit, this could be reduced to:

```python
            if getattr(self, "boto3_session", None) is None:
                # need bootstrapping
                self.boto3_session = boto3.Session(region_name=self.aws_default_region)
                self.autodetected_org_settings = {}
```